rebasehelper/build_helper.py

```diff
@@ -327,19 +329,21 @@ def build(cls, spec, sources, patches, results_dir, root=None, arch=None, **kwar
         # build SRPM
         srpm, cls.logs = cls._build_srpm(spec, sources, patches, results_dir)
 
+
         # build RPMs
         rpm_results_dir = os.path.join(results_dir, "RPM")
         with MockTemporaryEnvironment(sources, patches, spec, rpm_results_dir) as tmp_env:
             env = tmp_env.env()
             tmp_results_dir = env.get(MockTemporaryEnvironment.TEMPDIR_RESULTS)
-            rpms = cls._build_rpm(srpm, tmp_results_dir)
+            rpms = cls._build_rpm(srpm, tmp_results_dir, **kwargs)
```

Comment:
Then  please pass it explicitly instead of the whole dictionary, it is more readable.